app-services/secrets/README.md

```diff
@@ -48,49 +89,17 @@ Success! Disabled the secrets engine (if it existed) at: secret/
 Success! Enabled the kv secrets engine at: secret/
 ```
 
-#### Insecure mode
-
-When security is disabled, the secrets can be written to and are retrieved from the *Writable.InsecureSecrets* section of the configuration file. Insecure secrets and their paths can be configured as below.
-
-```toml
- [Writable.InsecureSecrets]    
-      [Writable.InsecureSecrets.NoPath]
-        Path = ''
-        [Writable.InsecureSecrets.NoPath.Secrets]
-          username = 'nopath-user'
-          password = 'nopath-pw'
-      
-      [Writable.InsecureSecrets.AWS]
-        Path = 'aws'
-        [Writable.InsecureSecrets.AWS.Secrets]
-          username = 'aws-user'
-          password = 'aws-pw'
-      
-      [Writable.InsecureSecrets.MongoDB]
-        Path = 'mongodb'
-        [Writable.InsecureSecrets.MongoDB.Secrets]
-          username = ''
-          password = ''
-```
-
-`NOTE: An empty path is a valid configuration for a secret's location  `
-
-#### Run StoreSecrets and GetSecrets
-
-##### StoreSecrets
-
-When running in secure mode, secrets for can be stored in the secret store (Vault) by making an HTTP `POST` call to `http://[host]:[port]/api/v1/secrets`.  
-
-*SecretsExample.postman_collection.json* contains Postman requests to store secrets to using the */secrets* API route. 
-
-If in secure mode, execute the Store Secrets Postman requests to push secrets to Vault.
+5. Verify that *SecretStore* and *SecretStoreExclusive* configuration values are correct.
+6. Run the secrets service: `go run main.go`
 
-If running in insecure mode, configure the secrets in consul or in the config yaml file (as described [above](#Insecure mode)).
 
-##### GetSecrets
 
-*SecretsExample.postman_collection.json* contains a Postman request to get secrets by triggering a CoreData event to the application service. 
+## Storing and Getting Secrets
 
-Execute the *CoreData Event Trigger (Random-Float-Device)* request and view the secrets in the application's console.
+1. In Postman, execute the `Store Secrets` request in the Postman collection *SecretsExample.postman_collection.json*, to push secrets to Vault.
+2. Execute the `CoreData Event Trigger (Random-Float-Device)` request in the Postman collection *SecretsExample.postman_collection.json* and view the secrets in the application's console.
+   * This request triggers a CoreData event to the application service which causes execution of the pipeline function that calls the GetSecrets() API.
+3. View the service's logs to verify that the secrets were retrieved.
+   1. Running in docker: `docker logs app-service-secrets-example`
```

Comment:
this currently logs to console. It's just used to demonstrate that GetSecrets() works. Since this an example, one would have to modify it do something more useful.